src/observerClass.js

```diff
@@ -0,0 +1,159 @@
+import { PureComponent, Component } from "react"
+import { createAtom, _allowStateChanges, Reaction, $mobx } from "mobx"
+import { useStaticRendering as useStaticRenderingLite } from "mobx-react-lite"
+
+import { newSymbol, shallowEqual, setHiddenProp, patch } from "./utils/utils"
+
+let isUsingStaticRendering = false
```

Comment:
Do you actually need to keep separate state for this? Why not reuse the variable from the Lite directly and export that setter directly as well?

Reply:
Because an imported variable is not bound to its original definition (that is, changes won't be reflected in it)

Reply:
I don't think it should matter. The Lite exports both setter and getter. It does not export the actual variable. For consistency, the same should be exported here as well imo.

Reply:
ah missed that, will change!

Reply:
Done

Reply:
Nit: The observerClass should only consume `isUsingStaticRendering`, but both should be exported directly from `index.js` without extra wrapping.

Reply:
Fixed